generic/tests/cfg/boot_vm_in_hugepage.cfg

```diff
@@ -10,8 +10,8 @@
     pre_command_noncritical = yes
     pre_command = "echo 3 > /proc/sys/vm/drop_caches"
     variants:
-        - 2M_hugepage:
-            expected_hugepage_size = 2048
+        - @default:
```

Comment:
1. I met negative memory value problem, so I can not to have test for this case.
2. there indeed is a default value for 'expected_hugepage_size':
            self.expected_hugepage_size = int(
                params.get("expected_hugepage_size", 0))

But it the expected_hugepage_size > 0, which mean you have set a hugepage size, there should be a hugepage check.
        if self.expected_hugepage_size:
            self.check_hugepage_size_as_expected()
3. IMO, If we want to avoid some invalid hugepage size problem, it should be done in the framework instead of implementing it in the case.
4. For the problem we met now, we could set filters in different variants.

Thanks.